riscv-target/opentitan/device/rv32imc/link.ld

```diff
@@ -0,0 +1,55 @@
+/* Copyright lowRISC contributors.
+   Licensed under the Apache License, Version 2.0, see LICENSE for details.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+OUTPUT_ARCH( "riscv" )
+ENTRY(_start)
+
+/* required to correctly link newlib */
+GROUP( -lc -lgloss -lgcc -lsupc++ )
```

Comment:
Can you format these parenthetical arguments cuddled to the braces `("riscv")`, `(-lc`, etc

Reply:
fixed